src/core/utils/ErrorUtils.ts

```diff
@@ -51,6 +51,10 @@ export const effectsErrorTextValue = function(user: User, language: string, play
  * @param reason
  */
 export async function replyErrorMessage(interaction: CommandInteraction, language: string, reason: string): Promise<void> {
+	if (interaction.replied) {
+		await interaction.editReply({embeds: [new DraftBotErrorEmbed(interaction.user, interaction, language, reason)]});
+		return;
+	}
```

Comment:
mais du coup ça va pas effacer l'ancienne reply ?? si le but c'est d'ajouter une sécurité why not mais on est d'accord que en théorie c'est du code qui devrais jamais être executé l'interieur de ce if ? 

Dans ce cas perso je suis plutot pour que le bot crash

Reply:
enfin genre c'est mieux de throw une exception dans les logs plutot que de faire un truc chelou coté utilisateur IMO